api-gateway/src/routes/apiGatewayRoutes.js

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import routerLogin from './routerLogin.js';
+import { authBearer } from'../middlewares/auth.js';
+import { createProxyMiddleware } from "http-proxy-middleware";
+
+const routes = (app) => {
+  
+  app.use('/api/users', createProxyMiddleware({ target: 'http://localhost:3003/', changeOrigin: true }));
```

Comment:
Se o api gateway for executado via docker-compose juntamente com os outros serviços, essa referência a localhost irá funcionar?